### PR TITLE
update fundamental styles library version in angular example 0.8.1

### LINKED ISSUE #1229: Update Fundamental Library in Angular example
- Refactor outdated components (for example modals).
- change the alerts --> message-strip
- change the loading spinner
- list-group updated but styled according to the needs pf Luigi
- check selects (https://sap.github.io/fundamental-styles/components/form.html)

### PR DESCRIPTION
**Pull request by UlianaMunich**
Fixes #1229 
- updated modal layout for Angular example
- fixed issues with fonts and icons
- removed couple of not needed badges
- updated the re-login button
- update the header styling

### CODE REVIEW

**Comment by marynaKhromova** [comment]
fonts

button

bar

**Comment by UlianaMunich** [comment]
@marynaKhromova added fonts to angular example and to components under the tests, please check. The rest should look good now.

File: website/fiddle/README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/6e36180d-2486-4daf-80a5-90a0ab363992/deploy-status)](https://app.netlify.com/sites/relaxed-kowalevski-49f468/deploys)
 
 Luigi fiddle, deployed via netlify and accessible at https://fiddle.luigi-project.io
+Go to the [Luigi Fiddle](https://fiddle.luigi-project.io) site to see Luigi in action and configure a sample application.
+
+### Code formatting for contributors
```
Comment (alexandra-simeonova) [inline website/fiddle/README.md:6]:
```suggestion
Luigi Fiddle, deployed via Netlify and accessible at https://fiddle.luigi-project.io.
The Fiddle allows you to see Luigi in action and configure a sample application.

### Install Luigi Fiddle on your machine
```

Just a small comment on the readme 🙂

File: test/e2e-test-application/src/app/project/modal/modal.component.html [hunk 2]
```diff
@@ -1,38 +1,47 @@
 <div
   *ngIf="modalActive"
-  class="fd-ui__overlay fd-overlay fd-overlay--modal"
+  class="fd-dialog fd-dialog--active fd-dialog-docs-static"
```
Comment (marynaKhromova) [inline test/e2e-test-application/src/app/project/modal/modal.component.html:3]: `fd-dialog-docs-static` is redundant

File: test/e2e-test-application/src/logout.html [hunk 3]
```diff
@@ -8,9 +8,12 @@
...
   <meta name="description" content="">
   <meta name="keywords" content="logout">
-  <link rel='stylesheet' href='/fundamental-styles/fundamental-styles.css'>
-
+  <link rel="stylesheet" type="text/css" href="/fundamental-styles/fundamental-styles.css">
+  <link rel="stylesheet" type="text/css" href="styles.css">
   <style>
+    body {
+      margin: 0;
+    }
```
Comment (marynaKhromova) [inline test/e2e-test-application/src/logout.html:16]: this can be removed since we have it already in styles.css

**Pull request merged**